command/server/config.go

```diff
@@ -366,15 +368,28 @@ func ParseConfig(d string, logger log.Logger) (*Config, error) {
 		return nil, err
 	}
 
-	if o := list.Filter("backend"); len(o.Items) > 0 {
-		if err := parseBackends(&result, o); err != nil {
-			return nil, fmt.Errorf("error parsing 'backend': %s", err)
+	// Look for storage but still support old backend
+	if o := list.Filter("storage"); len(o.Items) > 0 {
+		if err := parseStorage(&result, o, "storage"); err != nil {
+			return nil, fmt.Errorf("error parsing 'storage': %s", err)
+		}
+	} else {
+		if o := list.Filter("backend"); len(o.Items) > 0 {
+			if err := parseStorage(&result, o, "backend"); err != nil {
+				return nil, fmt.Errorf("error parsing 'backend': %s", err)
```

Comment:
Hey @jefferai 

Do you think we should print out a warning here (and below) telling folks that it's been renamed and will be removed in a future version?